packages/osd-ui-shared-deps/theme.ts

```diff
@@ -30,20 +30,21 @@
 
 // ToDo: Use `THEME_SOURCES` from `src/core/server/rendering/views/theme` to generate the logic below.
 import LightTheme from '@elastic/eui/dist/eui_theme_light.json';
+import { resolveThemeTag, resolveThemeVersion } from './theme_config';
 
 const globals: any = typeof window === 'undefined' ? {} : window;
 
 export type Theme = typeof LightTheme;
 
 // in the OpenSearch Dashboards app we can rely on this global being defined, but in
 // some cases (like jest) the global is undefined
-export const tag: string = globals.__osdThemeTag__ || 'v8light';
-export const version = tag.startsWith('v7') ? 7 : 8;
```

Comment:
This would also be a breaking change.

Reply:
How is this breaking?

Reply:
Currect; only the version which was a number is now not a number. A simple way out would be to expose `version` as a number that we don't use but create a new `versionString` or anything you like for the future and then mark version as deprecated.